packages/webapp/src/components/Form/InputBase/InputBaseLabel/styles.module.scss

```diff
@@ -13,11 +13,12 @@
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
 
-.labelContainer {
+ .labelContainer {
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-start; /* Keep items together */
   min-height: 20px;
   position: relative;
+  white-space: nowrap; /* Prevent wrapping */
 }
```

Comment:
The original CSS for this class looks fine to me.
I wonder why `tooltipIconContainer` has a width of `100%`, which is causing the issue. Removing all the styles from tooltipIconContainer fixes the issue, but could this break anything else?

I would ask Navdeep for his inputs so that we can fix it properly.